src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee

```diff
@@ -108,6 +110,10 @@ DashboardAppsDockCtrl = ($scope, $cookies, $uibModal, $window, MnoeOrganizations
           $scope.apps = MnoeAppInstances.appInstances
       )
 
+  # Hide the dock if marketplace is disabled and there is not app linked
+  $scope.displayDock = ->
+    return if !$scope.isMarketplaceEnabled && (!$scope.apps? || $scope.apps.length == 0 ) then false else true
```

Comment:
Oh I missed this one

```coffescript
$scope.displayDock = ->  
  $scope.isMarketplaceEnabled || ($scope.apps? && $scope.apps.length > 0)
```

* avoid `if condition then true else false`
* return are implicit in coffescript